chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1170,9 +1170,6 @@ impl PeerManagerActor {
         for peer_id in to_unban {
             if let Err(err) = self.peer_store.peer_unban(&peer_id) {
                 error!(target: "network", message = "Failed to unban a peer", ?err);
-                // TODO: Do we really want to return?
```

Comment:
can we have tests to cover these error cases ?

Reply:
This PR is fixing a bug - so we should have a test case, that 'fails' before this fix and 'works' afterwards.

(even if this is 'just' a unittest that simulates database read/write failure)

Reply:
@mm-near Currently, we don't have a good way to test triggers, nor simulate database failures.

Besides what do we event want to test? 
a) That this function continues to run, and there is no return?
The code has been now rewritten, we can see that there is no "return", that can be easily seen.
b) Test that trigger works. 
Triggers needs to be rewritten to be unit testable. I have a plan to do it, but that's a outside of scope of this  pr.